packages/dashboards/src/lib/components/timeseries-widget/chart-presets/xy-chart/xy-chart.component.less

```diff
@@ -0,0 +1,48 @@
+@import (reference) "@nova-ui/bits/sdk/less/nui-framework-variables.less";
+@import (reference) "@nova-ui/bits/sdk/less/mixins.less";
+
+@nui-legend-deemphasized-opacity: 0.3;
+@nui-legend-series-content-margin: 3px;
+
+.legend-container {
+    .legend-description-container {
+        & .legend-description {
+            line-height: @nui-line-height-default;
+            .text-overflow(ellipsis);
+            padding-left: @nui-space-xs;
+            padding-right: @nui-space-xs;
+            min-width: 130px;
+
+            &-primary {
+                font-weight: @nui-font-weight-semibold;
+                &:not(.nui-text-link-small) { 
+                    .setCssVariable(color, nui-color-text-default);
+                }
+                &:not(:last-child) {
+                    margin-bottom: -@nui-legend-series-content-margin;
+                }
+            }
+
+            &-secondary {
+                &:not(.nui-text-link-small) { 
+                    .setCssVariable(color, nui-color-text-secondary);
+                }
+                &:not(:first-child) {
+                    margin-top: -1px;
+                }
+            }
+        }
+    }
+
+    &:not(:hover)&.state-hidden {
+        .legend-description-container {
+            opacity: @nui-legend-deemphasized-opacity;
+            .legend-description-primary {
+                .setCssVariable(color, nui-color-text-default);
```

Comment:
Do we know whether we're supposed to remove the link styling for hidden series? I would think if you can still click it to navigate somewhere, we should keep the link styling, no?